app/src/main/res/values/strings.xml

```diff
@@ -799,6 +799,17 @@ Before uploading your changes, the app checks with a &lt;a href=\"https://www.we
     <string name="quest_board_type_map_title">Is it just a map and only a map?</string>
     <string name="quest_board_type_map_description">If the board is about a particular topic, please specify it, regardless of whether it also includes a map. For example a public transport board may have a bus route map. If the topic is not listed among the available answers, please consider leaving a note instead.</string>
 
+    <string name="quest_boat_rental_title">Which type of boats are for rent here?</string>
+    <string name="quest_boat_rental_motorboat">Motorboat</string>
+    <string name="quest_boat_rental_jetski">Jetski</string>
+    <string name="quest_boat_rental_sailboat">Sailboat</string>
+    <string name="quest_boat_rental_pedalboat">Pedalboat</string>
+    <string name="quest_boat_rental_kayak">Kayak</string>
+    <string name="quest_boat_rental_canoe">Canoe</string>
+    <string name="quest_boat_rental_houseboat">Houseboat</string>
+    <string name="quest_boat_rental_dinghy">Dinghy</string>
```

Comment:
Is that the same as a rowboat? Maybe call it "Dinghy / Rowboat" then

Reply:
I'd even go for `rawboat` alone. As noted above:

> I'd rather use [rowboat_rental](https://wiki.openstreetmap.org/wiki/Key:rowboat_rental)=`yes/no` instead of `dinghy_rental=yes/no` to avoid ambiguity ([Dinghy](https://en.wikipedia.org/wiki/Dinghy) may use motor instead of oars); especially as both tags (while documented) are not really used currently (single-digit uses worldwide).

IOW, rowboat is a subtype of dingy which uses oars (and not a motor) for moving.

Reply:
Resolved with 435b6c7b552a89b61281b81582df87a42acd75d5